.github/workflows/publish_deploy_image.yaml

```diff
@@ -2,9 +2,9 @@ name: Create and publish a Docker image to ghrc.io and deploy to openshift
 
 on:
   push:
-    branches: [ main ]
```

Comment:
Looks like I will have to align with @BryanGK on your Prettier settings when I get back